server/balancer_test.go

```diff
@@ -102,6 +102,14 @@ func (s *testBalancerSuite) updateStore(c *C, clusterInfo *clusterInfo, storeID
 	c.Assert(ok, IsTrue)
 }
 
+func (s *testBalancerSuite) updateStoreState(c *C, clusterInfo *clusterInfo, storeID uint64, state metapb.StoreState) {
+	storeInfo := clusterInfo.getStore(storeID)
+	storeInfo.store.State = state
+	clusterInfo.addStore(storeInfo.store)
```

Comment:
`updateStoreStatus` only update the `pdpb.StoreStats` part, but I need to update the `metapb.Store`.